fractal-theme/assets/js/scripts.js

```diff
@@ -1,16 +1,15 @@
-import colorHexMatches from './src/colors';
-import lastBuild from './src/overview';
-
-const modules = [
-  {selector: '.frctl-example', init: colorHexMatches},
-  {selector: '.frctl-overview', init: lastBuild},
-];
-
-modules.forEach((module) => {
-  const components = document.querySelectorAll(module.selector);
-
-  for (let i = 0; i < components.length; i++) {
-    const component = components[i];
-    module.init(component);
-  }
-});
+import colorHexMatches from './src/colors';
+
+const modules = [
+  {selector: '.frctl-example', init: colorHexMatches},
+  {selector: '.frctl-overview', init: lastBuild},

```

Comment:
I guess this line of code is now obsolete?

Reply:
Fixed with latest commit.